### PR TITLE
Add more filters

### PR DESCRIPTION
**Pull request by ZachCMP**
https://www.pivotaltracker.com/story/show/185334603
https://www.pivotaltracker.com/story/show/185274189

Relies on frontend PR: https://github.com/greenriver/hmis-frontend/pull/308

### CODE REVIEW

File: drivers/hmis/app/models/hmis/hud/enrollment.rb [hunk 1]
```diff
@@ -93,13 +93,21 @@ class Hmis::Hud::Enrollment < Hmis::Hud::Base
 
   scope :not_in_progress, -> { where.not(project_id: nil) }
 
-  scope :with_project_type, ->(project_types) do
-    wip_enrollments = joins(wip: :project).where(p_t[:project_type].in(project_types)).pluck(wip_t[:source_id])
-    nonwip_enrollments = joins(:project).where(p_t[:project_type].in(project_types)).pluck(:id)
+  scope :with_matching_projects, ->(query) do
```
Comment (gigxz) [inline drivers/hmis/app/models/hmis/hud/enrollment.rb]: Nice, I like this approach. Maybe it should be called `:in_projects_where` or `with_projects_where`?

Comment (ZachCMP) [reply drivers/hmis/app/models/hmis/hud/enrollment.rb]: Agreed. I chose `with_projects_where`

File: drivers/hmis/app/graphql/types/hmis_schema/assessment.rb [hunk 2]
```diff
@@ -13,6 +13,8 @@ class HmisSchema::Assessment < Types::BaseObject
 
     available_filter_options do
       arg :roles, [Types::Forms::Enums::AssessmentRole]
+      arg :project_types, [Types::HmisSchema::Enums::ProjectType]
+      arg :projects, [ID]
```
Comment (gigxz) [inline drivers/hmis/app/graphql/types/hmis_schema/assessment.rb]: Can you un-pluralize these (like you did for service)? `project_type` and `project`.

I opened a PR to do a whole sweep to make them consistent https://github.com/greenriver/hmis-warehouse/pull/3132

Comment (ZachCMP) [reply drivers/hmis/app/graphql/types/hmis_schema/assessment.rb]: Fixed

**Review by gigxz — APPROVED** [review]

**Review by gigxz — APPROVED** [review]

**Comment by gigxz** [comment]
HMIS test suite is passing locally; merging

**Pull request merged**